src/usb/debug.c

```diff
@@ -438,20 +446,18 @@ flashcarts.
             debug_thread_usb(&msg);
         #endif
     }
-
-
+    
+    
     /*==============================
         _debug_assert
         Halts the program (assumes expression failed)
         @param The expression that was tested
         @param The file where the exception ocurred
         @param The line number where the exception ocurred
     ==============================*/
-
+    
     void _debug_assert(const char* expression, const char* file, int line)
     {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
```

Comment:
New changes look essentially nonexistent, so I take it these throw warnings again?